### ISSUE TITLE
Dashboard redesign

### ISSUE THREAD

**Issue by maparent** [topicquests/sensecraft#179]
We need to redesign the dashboard, and minimize many pages accordingly.
Proposal on [this page](https://github.com/topicquests/sensecraft/wiki/Dashboard-design) so it's easier for all to edit.

**Comment by maparent** [comment]
Note: depends on #159 wip.

**Comment by maparent** [comment]
No major objections with redesign, moved ahead with #159, we need to show the information now.

**Comment by maparent** [comment]
Not sure if it belongs here or in another task: The current home page should become an "about" page. The dashboard could thus be the current page. We could offer a link to the about page in the home page, maybe with more content if not logged in (or first visit.)
